schema/render.go

```diff
@@ -50,9 +50,12 @@ func (t *treeLevel) Type() parser.Type {
 	return parser.TypeUnknown
 }
 
-func (t *treeLevel) add(path paths.Path, property parser.Property) error {
+func (t *treeLevel) add(path paths.Path, property parser.Property, pruneBranch bool) error {
```

Comment:
I'm always a little nervous seeing these booleans; it it always hard to know what `true` or `false` means, for example:

```go
root.add(property.Path, property, false)
```

I'd suggest going with smth like

```
type Mode int
const (
	PruneBranch = Mode(iota)
	DontPruneBranch
)
```

I don't know if this snippet is correct, but you get the idea. Func calls would be much more explicit:


```go
root.add(property.Path, property, DontPruneBranch)
```

Reply:
Sorry, this PR contained the wrong changes.
See https://github.com/cert-manager/helm-tool/pull/43 instead.